database/src/migrations/20220407000000_treatment_unit_description.ts

```diff
@@ -0,0 +1,23 @@
+import { Knex } from 'knex';
+
+const DB_SCHEMA = process.env.DB_SCHEMA;
+
+/**
+ * Removes the `description` column from the `treatment_unit` table, since
+ * descriptions associated with Treatments is no longer required.
+ *
+ * @export
+ * @param {Knex} knex
+ * @return {*}  {Promise<void>}
+ */
+export async function up(knex: Knex): Promise<void> {
+  await knex.raw(`
+		SET schema ${DB_SCHEMA};
```

Comment:
```suggestion
		SET schema '${DB_SCHEMA}';
```

Noticed the migration failed, not super obvious, but the schema has to be wrapped in single quotes.

Reply:
(setting the search_path doesn't require wrapping the values in quotes, because consistency..)

Reply:
The logs, if you are curious: https://console.apps.silver.devops.gov.bc.ca/k8s/ns/af2668-dev/pods/restoration-tracker-db-setup-dev-73/logs